sensum-pwa/src/blockchain/eth/Ethers.js

```diff
@@ -0,0 +1,11 @@
+import { ethers } from 'ethers';
+
+export async function getNetwork(networkUrl) {
+  const provider = new ethers.providers.JsonRpcProvider(networkUrl);
```

Comment:
IMHO, is better to repeat this line than try to create a binding for the provider (given `ethers` is a stateful library)